tensorrt_llm/evaluate/audio_asr.py

```
@@ -0,0 +1,515 @@
+
+        prompt = trtllm_apply_chat_template(
+            model_type=input_context.model_type,
+            tokenizer=llm.tokenizer,
+            processor=input_context.processor,
+            conversation=conversation,
+            add_generation_prompt=True,
+            mm_placeholder_counts=mm_placeholder_counts_by_message,
+            chat_template_kwargs=input_context.chat_template_kwargs,
+        )
+
+        input = {"prompt": prompt}
```

Ruff (0.15.12)

[error] Variable `input` is shadowing a Python builtin

(A001)